### ISSUE TITLE
Support role-mentions othere than @here and @everyone

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [radiantly/live-countdown-bot#2]
Hi! We avoid @here and @everyone due to the non-granular, difficult-to-opt-out-of nature thereof, and instead have a @Ping role for the entire server, as well as more specific @Group roles for different interests.

At the moment, we can `!countdown TIMECODE @Gamers` or something like that, but that highlights the group only *when the countdown is made*, not later on when it triggers — ideally, I'd like to be able to highlight when-made, when-triggered, both, or neither, for any group on the server

Perhaps using Discord's "escape" functionality?

```
!countdown \@Gamers 11:59 PM EST # just notifies them when it lands
Hey @Gamers, playing at !!countdown \@Gamers 11:59 PM EST!, kay? # notifies when creating, and when it lands
```

**Comment by radiantly** [comment]
Hello! Thanks for taking the time to write out such a thoughtful suggestion!
This seems like a really useful feature, I'll get around to implement it when I get time :)

However, something to note is that `\@Gamers` pings the Gamers role anyway, so it cannot be used to notify only when the countdown is done.

**Comment by ELLIOTTCABLE** [comment]
Perhaps DM support?

```
# !countdown #noisy @Gamers 11:59 PM EST
!countdown \<#706585477351735332> \<@&724744022111027230> 11:59 PM EST
```

idk if the copy-paste-and-escape functionality is accessible for less-technical users, tho.

**Comment by radiantly** [comment]
An approach I'm considering is the following:
```
!countdown tag Gamers 11:59 PM EST #silent
!countdown tag @Gamers 11:59 PM EST #noisy
```
On the server side I can probably retrieve the role id based on the role name provided.

**Comment by ELLIOTTCABLE** [comment]
Awesome; I actually discovered that this was implemented by accident!

It looks like there *might* be a bug when the role name includes exclamation points? With our role named "Rock! And! Stone!", all of the following triggered errors:

<img width="578" alt="Screen Shot 2021-02-13 at 14 22 04" src="https://user-images.githubusercontent.com/200/107860776-e2de2300-6e06-11eb-859a-60c771f0bbca.png">

Not a huge deal, just thought you might like to know! <3
